Documentation/source/Updating to 4.0.md

```diff
@@ -1,2 +1,49 @@
 Updating to 4.0
 =============================
+
+## What's new?
+---
+
+### Multi-page support
+Documents with multiple pages are now supported, being possible to capture several pages to get the analysis results. To enable it (disabled by default) set the `GiniConfiguration.multipageEnabled` property to `true`.
+Besides, when using the Component API, the `MultipageReviewViewController` component is used to handle one or several documents (see the Example app for implementation details).
+
+### Networking implementation
+The Screen API now offers the option to include all the networking analysis logic, only being necessary to provide a `GiniVisionResultsDelegate` when initializing the Screen API in order to get the analysis results.
+If you want to migrate your Screen API implementation to this one, just remove your current implementation of the `GiniVisionDelegate` and implement the `GiniVisionResultsDelegate`. You have had to add the additional line in your `Podfile` before, as pointed out in the [Integration guide](integration.html).
+
+### Custom photos gallery
+A custom photos gallery picker has been designed, which unlike the native `UIImagePickerController` allows to select multiple photos at the same time, to start caching before showing it (no delays when showing the first time) and some additional customization.
```

Comment:
Can you mention the name of the class?

Reply:
I don't want to mention it since it is not pubic. But I can point out that it is part of the `DocumentPickerCoordinator`.